lib/ansible/modules/windows/win_domain.ps1

```diff
@@ -79,7 +79,7 @@ try {
 } catch [System.DirectoryServices.ActiveDirectory.ActiveDirectoryObjectNotFoundException] {
     Add-Warning -obj $result -message "Error during forest query initialization: $($_.Exception.Message)"
 } catch [System.DirectoryServices.ActiveDirectory.ActiveDirectoryOperationException] { i
```

Comment:
any idea what the 'i' at the end of L81 is for? - it looks like it might be unintended to me.

Reply:
I appreciate it was already there before!